### PR TITLE
[Code Cleanup] Use case instead of elseif chain

### PR DESCRIPTION
**Pull request by terabyte25**

### CODE REVIEW

**Comment by Capostrophic** [comment]
terra no

**Comment by terabyte25** [comment]
what did I do wrong

**Comment by terabyte25** [comment]
oh wait the variable is supposed to be arg instead of gw

**Comment by psi29a** [comment]
Don't get it, so we're adding more lines by using a switch, but no default?

**Comment by psi29a** [comment]
Someone _really_ wants to make a commit. ;)

File: apps/openmw/mwscript/guiextensions.cpp [hunk 1]
```diff
@@ -189,15 +189,23 @@ namespace MWScript
                 }
                 else
                 {
-                    MWGui::GuiWindow gw = MWGui::GW_None;
-                    if (arg == 3)
+                    MWGui::GuiWindow gw;
+                    switch (arg) {
```
Comment (akortunov) [inline apps/openmw/mwscript/guiextensions.cpp]: Usually we place brackets to the separate line.

Comment (terabyte25) [reply apps/openmw/mwscript/guiextensions.cpp]: Right.

**Comment by akortunov** [comment]
So what we decided to do? As for me, the switch here is an overkill. Since this function is not widely used, I suppose a couple of elseifs should be enough here.

**Comment by psi29a** [comment]
This isn't really code clean up but adding more code for the sake of it.

I'm going to close this for now.

**Pull request closed without merging**